Data/FitnessTrackerContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FitnessTracker.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace FitnessTracker.Data
+{
+    public class FitnessTrackerContext : DbContext
+    {
+        public FitnessTrackerContext(DbContextOptions<FitnessTrackerContext> options) : base(options) { }
+
+        public DbSet<Log> Logs { get; set; }
+
+        public DbSet<Exercise> Exercises { get; set; }
+    }
```

Comment:
Look at the OnModelCreating override for the DB Context, you'll need to override and ensure your entities are configured within that so Foreign Keys and DB Constraints are set up properly.

Reply:
added OnModelingCreating